homeassistant/components/tuya/select.py

```diff
@@ -27,6 +27,15 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Light
+    # https://developer.tuya.com/en/docs/iot/categorydj?id=Kaiuyzy3eheyy
+    "dj": (
+        SelectEntityDescription(
+            key=DPCode.WORK_MODE,
+            name="Mode",
+            entity_category=EntityCategory.CONFIG,
+        ),
+    ),
```

Comment:
This is part of the light mode of an light, and should thus not be a select entity but needs to be handled by the light entity.